mvpa2/measures/anova.py

```diff
@@ -79,12 +79,12 @@ def _call(self, dataset):
         ul = targets_sa.unique
 
         na = len(ul)
-        bign = float(dataset.nsamples)
         alldata = dataset.samples
 
         # total squares of sums
         sostot = np.sum(alldata, axis=0)
         sostot *= sostot
+        bign = sostot.dtype.type(dataset.nsamples)
```

Comment:
this doesn't sound right -- if alldata.dtype is int, you will end up with int here, which would also point to the bug below that inplace division

```
$> python -c 'import numpy as np; a =np.arange(8).reshape((2, -1)); a = np.sum(a, axis=0); a*= a; print a.dtype.type; b=a.copy();  a /= 3; print a; print b/float(3)' 
<type 'numpy.int64'>
[ 5 12 21 33]
[  5.33333333  12.          21.33333333  33.33333333]
```
and I guess test isn't precise to care about those rounding effects